msctl

```diff
@@ -1407,7 +1407,7 @@ overviewer() {
             "color" : "aqua",
             "clickEvent" : {
               "action" : "open_url",
-              "value" : "'$MAPS_URL'/'$1'"
+              "value" : "'$MAPS_URL'"
```

Comment:
Why drop the world here?  As far as I know, we don't have an `index.html` generated for this location, so directory browsing would have to be turned on for this to work.